client/src/features/project/components/ProjectEntityHeader.tsx

```diff
@@ -20,48 +20,53 @@ import React from "react";
 
 import EntityHeader from "../../../components/entityHeader/EntityHeader";
 import type { EntityHeaderProps } from "../../../components/entityHeader/EntityHeader";
-import {
-  useGetProjectIndexingStatusQuery,
-  useProjectMetadataQuery,
-} from "../projectKgApi";
+import { useGetProjectIndexingStatusQuery } from "../projectKgApi";
 import { ProjectStatusIcon } from "./migrations/ProjectStatusIcon";
+import { useProjectMetadataQuery } from "../../projects/projectsKgApi";
 
 type ProjectEntityHeaderProps = EntityHeaderProps & {
   branch: string;
   projectId: number;
 };
 
 export function ProjectEntityHeader(props: ProjectEntityHeaderProps) {
-  const { fullPath } = props;
+  const { branch, devAccess, fullPath, gitUrl, projectId } = props;
 
-  const projectIndexingStatus = useGetProjectIndexingStatusQuery(
-    props.projectId,
-    {
-      skip: !fullPath || !props.projectId,
-    }
-  );
+  const projectIndexingStatus = useGetProjectIndexingStatusQuery(projectId, {
+    skip: !fullPath || !projectId,
+  });
 
-  const { data: projectInKgData } = useProjectMetadataQuery(
-    { projectPath: fullPath },
-    { skip: !fullPath || !projectIndexingStatus.data?.activated }
+  const projectMetadata = useProjectMetadataQuery(
```

Comment:
Nitpick: variable name for readability

```suggestion
  const projectMetadataQuery = useProjectMetadataQuery(
```